### PR TITLE
feat: add more text on line 3

### PR DESCRIPTION
**Pull request by lofi-nowhale**
Added a more text as a header on line 3 to respond to line 1.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1 +1,3 @@
-"Hello there!"
\ No newline at end of file
+"Hello there!"
+
+# "Hello there!"
```
Comment (vsanantone) [inline README.MD:3]: Well  thank you for the nice gesture.

Comment (vsanantone) [reply README.MD:3]: Well thank you for the nice gesture once again. :)

**Review by vsanantone — COMMENTED** [review]
blah

File: README.MD (hunk 1 above)
Comment (lofi-nowhale) [reply README.MD:3]: Thanks for completing this review!

**Pull request merged**